docs/_posts/2016-01-08-A-implies-B-does-not-imply-B-implies-A.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "(A => B) !=> (B => A)"
+author: jimfb
+---
+
+The documentation for `componentWillReceiveProps` states that `componentWillReceiveProps` will be invoked when the props change as the result of a rerender.  Some newbies assume this means "if `componentWillReceiveProps` is called, then the props must have changed", but that conclusion is logically incorrect.
```

Comment:
https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=define%3Anewbie

 > newbie - noun - an inexperienced newcomer to a particular activity.

Seems like the perfect word, though admittedly a little informal.  But I don't feel strongly, so I'll change it :P.